Crashed airliner

RarkGrames opened the pull request:
#### Summary
```SUMMARY: Content "Added crashed airliner location"```

#### Purpose of change
New location and other stuff to explore.

#### Describe the solution

- Adding the new location
- Adding metal seat furniture
- Adding the crash axe

Quality screenshots

Front part:
![01](https://user-images.githubusercontent.com/50421549/61910210-44dcec00-af34-11e9-8f0e-da3fa4813ab6.png)

Rear part:
![03](https://user-images.githubusercontent.com/50421549/61910285-7b1a6b80-af34-11e9-8b65-93d719fc0808.png)

Trail of the crash:
![04](https://user-images.githubusercontent.com/50421549/61910331-a309cf00-af34-11e9-969c-07e479c6f980.png)

Cargo deck:
![05](https://user-images.githubusercontent.com/50421549/61910369-bfa60700-af34-11e9-8203-ecdb82557f2c.png)

The top and an overall picture:
![06](https://user-images.githubusercontent.com/50421549/61910399-d8162180-af34-11e9-910f-9c514bea1205.png)

#### Additional context
The trail is too short, but I guess nobody really needs a 3x15 overmap special

John-Candlebury commented:
If you actually spawn monsters or debris on the 3x15 special it could actually be pretty cool.
Well thats if you actually feel like doing it

Anyways great work!

LaVeyanFiend commented:
https://www.amazon.com/Gemtor-D42-Crash-Axe/dp/B00M24C2U8

Is this what the crash axe is supposed to be?

Also, I'd love a recipe for it. :3

roaringjohn commented:
I think you need to lint, based on the errors from the checks.

Night-Pryanik commented:
Whoo-hooo! Really cool!

Night-Pryanik commented on data/json/overmap/overmap_special/specials.json:
```diff
@@ -5328,5 +5328,36 @@
+    "locations": [ "wilderness" ],
+    "city_distance": [ 6, -1 ],
+    "city_sizes": [ 0, 12 ],
+    "occurrences": [ 0, 4 ],
```
With this value it could spawn up to 4 crashed airliners on an overmap. That's a LOT.
```suggestion
    "occurrences": [ 0, 1 ],
```

tinukedaya commented:
Wow! That's some luxury airliner! Look at that legroom!

It looks awesome, but I have few notes to the design.
- Don't leave the empty space between the rows. At least in the aft part of the plane, where the economy class is... ;) 
- It also looks more like quite a successful emergency landing than a crash. You might want to wreck it a little more. 
- I would probably put the cargo deck on Z-0 and Passenger deck on Z+1. If the lower deck dig itself into the ground it would probably be completely destroyed. For access into the passenger deck we would then need to use a ladder, or there might be few places around the front part of the crash, where the ground would create a slope up (similar to what is around the LMOE shelters).  
- About the lower deck. There are no corridors around the storage areas. It's basically just big hollow place loaded either by containers which contain the bags/cargo or smaller planes, like this one seems to be, would not even have containers and everything is just loaded loose in the hold. After the crash the containers will probably be destroyed so you can just leave it as empty space with trash/rubble/loot pilled up in the forward part of the hold.

l29ah commented:
Damn i can't mine seatbelts from it.

RarkGrames commented:
> 
> 
> If you spawn monsters or debris on the 3x15 special it could actually be pretty cool.

I will see if 5 more tiles length is enough to look good. It is mostly copy-paste work anyway. I'm just wondering if it messes with the mapgen.

RarkGrames commented:
> 
> 
> https://www.amazon.com/Gemtor-D42-Crash-Axe/dp/B00M24C2U8
> 
> Is this what the crash axe is supposed to be?
> 
> Also, I'd love a recipe for it. :3

Yes, this type of axe. The recipe will fit in the firefighter book perfectly.

RarkGrames commented:
> 
> 
> Damn i can't mine seatbelts from it.

I don't know if making a separate "airplane seat" furniture (that is used only here) is a good solution. Dropping short ropes on them looks ugly. I will try to figure out which one is the lesser of two evils.

RarkGrames commented:
@tinukedaya First I tried leaving no spaces between the seats, and it looked like the chair section of an Ikea warehouse. It had no sense of rows of seats.

The plane is intact because the blockiness of tiles makes very hard to recognize what this thing even is if you stumble upon it randomly in the woods. Especially that you can't see the tail on the ground Z level. Originally I wanted it to be in several pieces, but for readability I left it like this.

The front half of the lower deck is destroyed (the screenshot doesn't do a good job showing the proportions), it was ground down gradually during the slide, and now only the second half is intact. The side doors would be around 1 meter above ground as the fuselage is wedged into the ground. I tried to illustrate this using the dirt mounds that you can use to climb into the fuselage.
For the cargo deck, [this](http://lh6.ggpht.com/_yrfij5hMaRw/SX4N81Qs3mI/AAAAAAAABg8/P0tSZ_sKOPI/Flight+Museum+-+B707+Section.JPG) was the plan.

kevingranade commented:
> The plane is intact because the blockiness of tiles makes very hard to recognize what this thing even is if you stumble upon it randomly in the woods.

That's expected,  plane crashes aren't particularly recognizable, and "hey guys, what's this weird metal wreckage in the woods?" Is a great outcome.

anothersimulacrum commented on data/json/items/tools.json:
```diff
@@ -1506,6 +1506,27 @@
+    "material": [ "steel" ],
+    "techniques": [ "WBLOCK_1" ],
+    "weight": 907,
+    "volume": 4,
```
Volumes should be specified as a string.
```suggestion
    "volume": "1 L",
```

tinukedaya commented:
> @tinukedaya First I tried leaving no spaces between the seats, and it looked like the chair section of an Ikea warehouse. It had no sense of rows of seats.

well, seeing as everywhere in the game the seats rows are pictured one tile after another, I would do it here as well for consistency. As I mentioned, do it just in the aft part of the plane, it's simulate the very crowded economy class, while the extra free rows in front are for business. 

> The plane is intact because the blockiness of tiles makes very hard to recognize what this thing even is if you stumble upon it randomly in the woods. Especially that you can't see the tail on the ground Z level. Originally I wanted it to be in several pieces, but for readability I left it like this.

I don't think that would be a problem. just break it in three parts, leave few empty tiles between them, offset/rotate them a little so they are not in one line and voila... looks more "crashy"

> The front half of the lower deck is destroyed (the screenshot doesn't do a good job showing the proportions), it was ground down gradually during the slide, and now only the second half is intact. The side doors would be around 1 meter above ground as the fuselage is wedged into the ground. I tried to illustrate this using the dirt mounds that you can use to climb into the fuselage.
> For the cargo deck, [this]() was the plan.

The green thing in the reference picture is a container inside the hold. it would be destroyed during the crash most likely. [The hold looks like this.](https://i.stack.imgur.com/Q1L0G.jpg). Loaded with containers [like this](https://qph.fs.quoracdn.net/main-qimg-60708eb7b39c812d87cd6ca3e1087c3b). Smaller planes don't even have containers and are loaded [like this](https://i.stack.imgur.com/Nf6Y7.jpg)
So what you really want to have is just empty place filled with rubble and loot.

RarkGrames commented:
Reworked the cargo hold:
![04](https://user-images.githubusercontent.com/50421549/62007115-0e78ba00-b14a-11e9-918b-308abb7e13ef.png)

Broke down the front section and wrecked the nose:
![01](https://user-images.githubusercontent.com/50421549/62007122-22bcb700-b14a-11e9-8bcb-93df7fbd2005.png)

Added 7 om tiles of trail:
![02](https://user-images.githubusercontent.com/50421549/62007133-32d49680-b14a-11e9-97fb-d12cf67997ab.png)

![03](https://user-images.githubusercontent.com/50421549/62007134-36681d80-b14a-11e9-8d1f-d03ff17dc4bf.png)

RarkGrames commented:
> well, seeing as everywhere in the game the seats rows are pictured one tile after another, I would do it here as well for consistency.

I've checked out all the similar maps with rows of seats or benches I could find. I've found five of them: movie theater, church, cathedral, football field and stadium. All of them had a row of empty tiles between the two rows of seats.

RarkGrames commented:
> 
> 
> Damn i can't mine seatbelts from it.

Changed the seat furniture to yield a short rope.

tinukedaya commented:
> > well, seeing as everywhere in the game the seats rows are pictured one tile after another, I would do it here as well for consistency.
> 
> I've checked out all the similar maps with rows of seats or benches I could find. I've found five of them: movie theater, church, cathedral, football field and stadium. All of them had a row of empty tiles between the two rows of seats.

I meant seat rows in vehicles. not in buildings. In buildings you usually do not fight over leg space. Check bus, military truck, SWAT truck. Or any car actually. 

Cargo hold looks much better now. maybe put in some more dirt tiles toward the forward area. it would be badly damaged there. 

Also like the offset of the front part. do something similar for the tail and wreck the wing a litte as well. it will hardly survive in one piece.

paulenka-aleh commented:
Not that a big deal at this stage but crashed airliner looks like good source of steel. However steel isn't used to build passenger airliners, aluminium is used instead. It gotta be fixed someday if not now.

RarkGrames commented:
> 
> 
> Not that a big deal at this stage but crashed airliner looks like good source of steel. However steel isn't used to build passenger airliners, aluminium is used instead. It gotta be fixed someday if not now.

Good catch, although it's the same situation as the downed choppers. I doubt that anyone will ever use all the steel from the walls, and if you need just a little bit to craft some tools or something, you will find at least some steel lying around the wreck. I could add aluminum walls, but planes are not made only from aluminum.

RarkGrames commented:
For reference, filling the empty rows with seats looks like this:
![01](https://user-images.githubusercontent.com/50421549/62239953-2a908b80-b3d6-11e9-91b9-b0705842cab9.png)

tinukedaya commented:
> For reference, filling the empty rows with seats looks like this:

Now this looks more like an modern airliner ;) 
As I said, you can leave few rows as they were in the forward to make for the business class.

ZhilkinSerg commented:
Should probably be a map extra, but that could wait until #31682.

ZhilkinSerg commented on data/json/furniture_and_terrain/furniture-seats.json:
```diff
@@ -54,6 +54,37 @@
+    "looks_like": "f_armchair",
+    "symbol": "H",
+    "description": "An airplane seat with a seatbelt.",
+    "color": "gray",
```
```suggestion
    "color": "light_gray",
```

ampersand55 commented:
It would be cool it if had salvageable gas turbine engines its wings.
